main/calculations.py

```diff
@@ -0,0 +1,147 @@
+import requests
+import json
+from datetime import datetime
+from constance import config
```

Comment:
plik ze stałymi zwyczajowo nazywa się const lub constants.

Reply:
To nie jest plik - to biblioteka zewnętrzna (django-constance)